src/components/SingleArticle/SingleArticle.tsx

```diff
@@ -27,9 +27,19 @@ export const SingleArticle = ({ article }: { article: ArticleRecord }) => {
     day: '2-digit',
   })
   const publishedDateObj = new Date(publishedDate)
+
```

Comment:
wondering if we can handle this like we do labels and tags on this page, like:

`        {hero && (
          <img
            src={hero.url}
            alt="article hero graphic"
            className={styles.hero}
          />
        )}`

Reply:
I thought I tried this, but now that you mention it it was back when I had url as nullable in the types. I'll update the PR.

Reply:
that makes sense! yeah i figured it was left over from a previous restriction

Reply:
It's good catch, thanks for calling it out :)